addons/ui/flexiMenu/init.sqf

```diff
@@ -1,5 +1,19 @@
 #include "\x\cba\addons\ui\script_component.hpp"
 
+FUNC(keyDown) = CBA_fnc_flexiMenu_keyDown;
+FUNC(keyUp) = CBA_fnc_flexiMenu_keyUp;
+FUNC(menu) = CBA_fnc_flexiMenu_menu;
+FUNC(list) = CBA_fnc_flexiMenu_list;
+FUNC(getMenuDef) = CBA_fnc_flexiMenu_getMenuDef;
+FUNC(getMenuOption) = CBA_fnc_flexiMenu_getMenuOption;
+FUNC(menuShortcut) = CBA_fnc_flexiMenu_menuShortcut;
+FUNC(mouseButtonDown) = CBA_fnc_flexiMenu_mouseButtonDown;
+FUNC(highlightCaretKey) = CBA_fnc_flexiMenu_highlightCaretKey;
+FUNC(execute) = CBA_fnc_flexiMenu_execute;
+FUNC(add) = CBA_fnc_flexiMenu_add;
+FUNC(remove) = CBA_fnc_flexiMenu_remove;
+FUNC(setObjectMenuSource) = CBA_fnc_setObjectMenuSource;
```

Comment:
Now you moved them above the `isDedicated exitWith` thingy.

Reply:
Yeah. Because what happens if people call the funcs on server? Optimally they should do nothing I guess

Reply:
Leave me alone I don't have kids!